### ISSUE TITLE
doesnt work anymore ?

### ISSUE THREAD

**Issue by amit-handa** [kc-dot-io/node-google-alerts#6]
just tried it. doesnt work anymore I guess. hope I am not missing anything ?

**Comment by ankitsolanki** [comment]
You are right, doesnt work any more. Did you find an alternate solution?

**Comment by amit-handa** [comment]
hmmm, I dont remember much. last conclusion was that google had changed its alerts page. embedded more javascript into it. so that module doesnt work any more into it. may be we need to develop our own module on top of phantom ( or other similar frameworks) and achieve our target.

**Comment by slajax** [comment]
You are correct, the google alerts interface has changed and thus, thus won't work unless it has been updated for the new UX. I'll put something on the readme. Pull requests welcome!

**Comment by Muneem** [comment]
any other solution?Its not working

**Comment by kc-dot-io** [comment]
@Muneem unfortunately not. This is because the UX has changed. Since this script is simply a headless browser that simulates you manually clicking through, it should be really easy to update. If you would like to update it and push a PR, I'll review and merge.
